internal/utils/metrics/src/main/java/org/eclipse/ditto/internal/utils/metrics/instruments/timer/OnStopHandler.java

```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright (c) 2017 Contributors to the Eclipse Foundation
- *
- * See the NOTICE file(s) distributed with this work for additional
- * information regarding copyright ownership.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License 2.0 which is available at
- * http://www.eclipse.org/legal/epl-2.0
- *
- * SPDX-License-Identifier: EPL-2.0
- */
-package org.eclipse.ditto.internal.utils.metrics.instruments.timer;
-
-import java.util.function.Consumer;
-
-/**
- * Contains {@code stoppedTimerConsumer} to be invoked when a Timer stops.
- */
-final class OnStopHandler {
```

Comment:
what was the reason to delete this class? I think there's no need to have it as class but mabye as functional interface with `handleStoppedTimer` beeing the method?

Reply:
My personal opinion is that I don't like increasing abstraction.

Reply:
> I think there's no need to have it as class but mabye as functional interface with handleStoppedTimer beeing the method

Sounds like a good compromise 😃